filecoin-proofs/src/api/seal.rs

```diff
@@ -183,16 +200,23 @@ where
 
     // Load data tree from disk
     let data_tree = {
+        let tree_size =
+            get_tree_size::<<DefaultPieceHasher as Hasher>::Domain>(porep_config.sector_size);
+        let tree_leafs =
+            get_tree_leafs::<<DefaultPieceHasher as Hasher>::Domain>(porep_config.sector_size);
+
+        info!(
+            "seal phase 2: tree size {}, tree leafs {}, cached above base {}",
+            tree_size,
+            tree_leafs,
+            StoreConfig::default_cached_above_base_layer(tree_leafs)
+        );
         let config = StoreConfig::new(
             cache_path.as_ref(),
             CacheKey::CommDTree.to_string(),
-            DEFAULT_CACHED_ABOVE_BASE_LAYER,
```

Comment:
maybe we remove this from the merkletree lib, to avoid its usage entirely?

Reply:
Agreed, I'm working toward not exposing it anyway.